pkg/psalabelsyncer/podsecurity_label_sync_controller.go

```diff
@@ -284,19 +284,29 @@ func (c *PodSecurityAdmissionLabelSynchronizationController) syncNamespace(ctx c
 	nsCopy := ns.DeepCopy()
 
 	var changed bool
+	if nsCopy.Labels[psapi.EnforceLevelLabel] != string(psaLevel) || nsCopy.Labels[psapi.EnforceVersionLabel] != currentPSaVersion {
+		changed = true
+		if nsCopy.Labels == nil {
+			nsCopy.Labels = map[string]string{}
+		}
+
+		nsCopy.Labels[psapi.EnforceLevelLabel] = string(psaLevel)
+		nsCopy.Labels[psapi.EnforceVersionLabel] = currentPSaVersion
+	}
+
+	// cleanup audit and warn labels from version 4.11
+	// TODO: This can be removed in 4.13 and allow users set these as they wish
```

Comment:
if we own it, we own it.  We should keep control here.  If someone wants something different, they don't want our controller active.